torch_brain/dataset/dataset.py

```
@@ -0,0 +1,270 @@
+
+    Args:
+        data: The :class:`temporaldata.Data` object to modify.
+        path: The dot-separated path to the nested attribute (e.g., "session.id").
+        value: The value to set for the attribute.
+
+    Returns:
+        The modified data object (same instance, modified in-place).
+
+    Raises:
+        AttributeError: If any component of the path cannot be resolved.
+    """
```

Ruff (0.14.14)

[warning] f-string used as docstring. Python will interpret this as a joined string, rather than a docstring.

(B021)